game/classes/Player.ts

```diff
@@ -136,14 +136,90 @@ export class Player extends TileObject {
+        /* Change flow mode flow mode */
+        if (trigger.typeTrigger.includes("INF") && trigger.effectOn) {
+          main().multiplayer.game.level.flags = main().multiplayer.game.level.flags & (~ LevelFlags.FlowModeMask ) | LevelFlags.FlowAlways;
+          // unable to use it again
+          if (trigger.typeTrigger.includes("SNG")) {
+            trigger.effectOn = false;
+            this.curLevel.applyObstacleColour(trigger, 0xFF660000);
+          }
+        } else if (trigger.typeTrigger.includes("NRM") && trigger.effectOn) {
+          main().multiplayer.game.level.flags = main().multiplayer.game.level.flags & (~ LevelFlags.FlowModeMask ) | LevelFlags.FlowNormal;
+            if (trigger.typeTrigger.includes("SNG")) {
+              trigger.effectOn = false;
+              this.curLevel.applyObstacleColour(trigger, 0xFF660000);
+            }
+        } else if (trigger.typeTrigger.includes("NOF") && trigger.effectOn)  {
+          main().multiplayer.game.level.flags = main().multiplayer.game.level.flags & (~ LevelFlags.FlowModeMask ) | LevelFlags.FlowDisabled;
+            if (this.burningFlow)
+              this.parent["stopBurningFlowMusic"]();
+            this.burningFlow = false;
+            if (trigger.typeTrigger.includes("SNG")) {
+              trigger.effectOn = false;
+              this.curLevel.applyObstacleColour(trigger, 0xFF660000);
+            }
+        } 
+        /* set the value of flow points */
+        else if (trigger.typeTrigger.includes("STF")) {
+          var flowVal = +trigger.typeTrigger.slice(3);
+          this.flowPoints = flowVal;
+        } 
+        /* trigger that resets jump on collision */
+        else if (trigger.typeTrigger.includes("JMP")) {
+          this.jumpLevel = true;
+          this.yLove = 0;
+        /* Not fully working, keep for later
+      } else if (trigger.typeTrigger.includes("GRV")) {
+          var newG = +trigger.typeTrigger.slice(3);
+          if (newG * this.yGrav < 0) 
+            (this.parent as Game).skin.rotation = ((this.parent as Game).skin.rotation + 180)%360;
+          (this.parent as Game).player. += 150;
+          this.yGrav = newG;
+      */}
+        /* trigger that sets a random skin */
+        else if (trigger.typeTrigger.includes("SKN")) {
+          const game = this.parent as Game;
+          this.headType = Math.floor(Math.random() * 24);
+          this.handType = Math.floor(Math.random() * 24);
+          this.colour = Math.floor(Math.random() * 0xFFFFFF);
+          this.colour2 = Math.floor(Math.random() * 0xFFFFFF);
+          game.addSkin();
+        } 
+        /* trigger to emit beams */
+        else if (trigger.typeTrigger.includes("BEM")) {
+          (this.parent as Game).emitBeam(this.x,this.y,this.width);
+        } else if (trigger.typeTrigger.includes("COL")) {
+          this.curLevel.colorBG = trigger.typeTrigger.slice(3) == "RND" ? Math.random()*0xFFFFFFFF : +trigger.typeTrigger.slice(3);
```

Comment:
Treating it as hex color would be easier to use:
```suggestion
          this.curLevel.colorBG = trigger.typeTrigger.slice(3) == "RND" ? Math.random()*0xFFFFFFFF : (parseInt(trigger.typeTrigger.slice(3),16)||0);
```